Sources/Prometheus/MetricTypes/Summary.swift

```diff
@@ -1,8 +1,11 @@
+import NIOConcurrencyHelpers
+
 /// Default quantiles used by Summaries
 public var defaultQuantiles = [0.01, 0.05, 0.5, 0.9, 0.95, 0.99, 0.999]
 
 /// Label type Summaries can use
 public protocol SummaryLabels: MetricLabels {
+    /// Quantile
     var quantile: String { get set }
```

Comment:
Some other comment maybe? would this be "human readable description of the quantile" perhaps?